docs/grpc/c#.md

```diff
@@ -6,7 +6,6 @@ This section enumerates what you need to do to write a client that communicates
 ### Prerequisites
 
 * .Net Core [SDK](https://dotnet.microsoft.com/download)
```

Comment:
not using windows but do all supported windows versions have wsl?

Reply:
Is there a list of the Windows versions lnd supports? The releases page has x86, x64 and arm but no version is mentioned. WSL needs Windows version >= 10.